.env.development

```diff
@@ -2,7 +2,7 @@
 REACT_APP_GATEWAY=https://acc.api.top.amsterdam.nl/api/v1/
 
 # To login using grip:
-REACT_APP_AUTH_URL="https://auth.grip-on-it.com/v2/rjsfm52t/oidc/idp/authorize?response_type=code&scope=openid&client_id=d3d664c7-bb33-4bf0-b7c9-b8bdf1199b76&redirect_uri=http%3A%2F%2Flocalhost%3A3001%2Fauthentication%2Fcallback"
+REACT_APP_AUTH_URL="/authentication/callback?code=1234567890abcdefghijkl"
```

Comment:
If this is the only change needed.
I'd advice to create a .env locally and dont check it it in.

The .env should contain only this line.

Reply:
excuus pas ik weer terug aan

Reply:
was iets te snel met de commit